client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,15 +33,16 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
```

Comment:
@aduth I'm a bit confused, in #1370 I only moved the deprecated styles to the bottom of the css file—they'd previously been deprecated. Can we use the `is-borderless` style here instead of the deprecated styles?